artiq/master/worker_impl.py

```diff
@@ -286,6 +286,10 @@ def write_results():
             f["run_time"] = run_time
             f["expid"] = pyon.encode(expid)
 
+    def log_tuples():
+        with open("tuples.txt", "a") as f:
```

Comment:
The current directory of the experiment may be subject to change (see discussions in https://github.com/m-labs/artiq/issues/1747, https://github.com/m-labs/artiq/issues/1543).
Would be more future-proof to put it where the master writes other files such as ``last_rid.pyon``.